### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by stavoxnetworks** [1000001101000/acp-commander#11] [General]
This app saved two devices of mine. A terrastation, and a linkstation. Both would only go to emergency mode. Even with debug options available I could not get either to update. I ran ACP_commander on both in EM mode and both devices came to life. I cant thank you enough!

**Comment by smatterchoo** [comment]
me too, I used acp-commander to upgrade my linkstation 210 drive to an 8TB hdd.  thank you for this!

**Comment by 1000001101000** [comment]
Cheers!
